src/server/game/Garrison/Garrison.cpp

```diff
@@ -483,11 +478,7 @@ void Garrison::AddMission(uint32 garrMissionId)
     mission.PacketInfo.MissionRecID = garrMissionId;
     mission.PacketInfo.OfferTime = time(nullptr);
     mission.PacketInfo.OfferDuration = missionEntry->OfferTime;
-    mission.PacketInfo.StartTime = time_t(2254525440);
-    mission.PacketInfo.TravelDuration = missionEntry->TravelTime;
-    mission.PacketInfo.MissionDuration = missionEntry->Duration;
```

Comment:
Why are you removing those 3 lines ?

Reply:
Confirmed for unstarted missions, but are your sure about it even when mission is already started (MissionState != 0) ?

And then you need to assign it in Garrison::StartMission or they'll always be default values

Reply:
cant find any sniffs where sended this packet with State != 0